components/SbcSigninButton.vue

```diff
@@ -1,111 +1,79 @@
 <template>
-  <div>
-    <v-menu bottom width="300" transition="slide-y-transition">
-      <template #activator="{ on, attrs }">
-        <v-btn
-          large
-          class="button-login"
-          aria-label="log in"
-          v-bind="attrs"
-          v-on="on"
+  <v-menu bottom width="300" transition="slide-y-transition">
+    <template #activator="{ on, attrs }">
+      <v-btn
+        large
+        class="button-login"
+        aria-label="log in"
+        v-bind="attrs"
+        v-on="on"
+      >
+        <span>Log in to my BC Registries Account</span>
+        <v-icon class="mr-n1 ml-2">mdi-menu-down</v-icon>
+      </v-btn>
+    </template>
+
+    <v-card>
+      <v-card-title>Select login method</v-card-title>
+
+      <v-divider></v-divider>
+
+      <v-list dense>
+        <v-list-item
+          key="bcsc"
+          :href="
+            $config.authURL +
+            'signin/bcsc/' +
+            encodeURIComponent($config.baseURL) +
+            'dashboard'
+          "
         >
-          <span>Log in to my BC Registries Account</span>
-          <v-icon class="mr-n1 ml-2">mdi-menu-down</v-icon>
-        </v-btn>
-      </template>
-      <v-card class="menu-card pa-0">
-        <div>
-          <v-card-title class="menu-card-title pt-2 pb-2 pr-0"
```

Comment:
Again, there's (usually) no need to assign a class here, since Vuetify already assigns a classname to its components.